modules/configuring-firewall.adoc

```diff
@@ -5,16 +5,87 @@
 [id="configuring-firewall_{context}"]
 = Configuring your firewall for {product-title}
 
-Before you install {product-title}, you must configure your firewall to access
-Red Hat Insights.
+Before you install {product-title}, you must configure your firewall to grant access to the sites that {product-title} requires.
 
 .Procedure
 
-* Set your firewall to allow the following host names and ports on the outgoing
-network firewall:
+. Whitelist the following registry URLS:
 +
-----
-cert-api.access.redhat.com:443
-api.access.redhat.com:443
-infogw.api.openshift.com:443
-----
+[cols="3,4",options="header"]
+|===
+|URL | Function
+|`registry.redhat.io`
+|Provides core container images
+
+|`*.quay.io`
+|Provides core container images
+
+|`sso.redhat.com`
+|The `https://cloud.redhat.com/openshift` site uses authentication from `sso.redhat.com`
+|===
+
+. Whitelist any site that provides resources for a language or framework that your builds require.
```

Comment:
Do we want a "such as..." example to make this more concrete.  Like some NPM registry thing for Node?

Reply:
> Most looks good to me.
> Should we add *.apps.. into this whitelist?

@ironcladlou, what do you think?

Reply:
All default routes (e.g. console, auth, prometheus) sit behind that domain. Also, keep in mind the user can configure the ingress wildcard domain prefix at install-time (but *.apps is the default). Does that help? I'm not 100% up to speed on the context here. @danehans WDYT?